From: Florian Maurer <f.maurer@outlook.de>
Date: Thu, 23 Oct 2025 14:22:01 +0200
Subject: base-files: disable wiphy renaming

The phy renaming is currently only done when the wifi iface is used.
This is too late for gluon - disable it therefore, so that all targets use phyX as well.
This is currently needed for the mediatek-mt7622 target.

Still needed on openwrt-25.12: although upstream
commit 4b7323e ("netifd: always call setup for disabled radios") partially
addresses the issue, the phy name remains incorrect when radios are not
brought up through netifd (e.g. in Gluon config/setup mode).

The proper fix is pending in https://github.com/openwrt/openwrt/pull/23860

Signed-off-by: Florian Maurer <f.maurer@outlook.de>

diff --git a/package/base-files/files/lib/functions/uci-defaults.sh b/package/base-files/files/lib/functions/uci-defaults.sh
index 34005fe0fa46e2ac8d134bc3113558f9646e86f9..0ee47e75e5e19f26be37c4f1649881252f2fa62b 100644
--- a/package/base-files/files/lib/functions/uci-defaults.sh
+++ b/package/base-files/files/lib/functions/uci-defaults.sh
@@ -785,6 +785,7 @@ ucidef_set_poe() {
 }
 
 ucidef_add_wlan() {
+	return 0
 	ucidef_wlan_idx=${ucidef_wlan_idx:-0}
 
 	json_select_object wlan
